### ISSUE TITLE
Access constant reference

### ISSUE THREAD

**Issue by Socob** [JuliaPhysics/PhysicalConstants.jl#35]
As far as I can tell, the “reference” that’s defined for a constant and printed in the `show` method, e. g.
```julia
julia> PhysicalConstants.CODATA2018.G
Newtonian constant of gravitation (G)
Value                         = 6.6743e-11 m^3 kg^-1 s^-2
Standard uncertainty          = 1.5e-15 m^3 kg^-1 s^-2
Relative standard uncertainty = 2.2e-5
Reference                     = CODATA 2018
```
is not available anywhere and only exists within that `show` method. It would be nice to have a way to access that separately, e. g. via a newly defined function.

**Comment by giordano** [comment]
Agreed, PR for an accessor function welcome!  Side note, in hindsight I'm not particularly happy with the design of having singletons for which lots of magic methods are defined (which is totally inspired by `Base.@irrational`), using proper structures would be cleaner (and also better for time-to-first-print a constant).

**Comment by Socob** [comment]
PR created!

I don’t know if there are any drawbacks associated with having more information stored in a struct itself, but for now I didn’t opt to change the entire structure of the package 😉

**Comment by giordano** [comment]
Yeah, no worries, that's a larger undertake
